src/instruments/src/SD/Pages/Eng/Eng.tsx

```diff
@@ -0,0 +1,312 @@
+            } else {
+                setShouldPressurePulse(false);
+            }
+
+            if (displayedEngineOilQuantity <= OIL_QTY_LOW_ADVISORY) {
+                setQuantityAtOrBelowLow(true);
+            }
+
+            if (quantityAtOrBelowLow && displayedEngineOilQuantity >= OIL_QTY_LOW_ADVISORY + 2) {
+                setQuantityAtOrBelowLow(false);
+            }
+        } else {
+            setShouldPressurePulse(false);
+            setShouldQuantityPulse(false);
+        }
+
+        if (displayedEngineOilPressure <= OIL_PSI_VLOW_LIMIT) {
+            setPsiNeedleRed(true);
+        }
+        if (psiNeedleRed && displayedEngineOilPressure >= OIL_PSI_VLOW_LIMIT + 0.5) {
+            setPsiNeedleRed(false);
+        }
+    }, [flightPhase, engineOilQuantity, engineOilPressure]);
+
+    useEffect(() => {
+        if (displayedEngineOilTemperature >= OIL_TEMP_HIGH_ADVISORY) {
+            setShouldTemperaturePulse(true);
+        } else if (!tempBeenAboveAdvisory) {
+            setShouldTemperaturePulse(false);
+        }
+
+        if (displayedEngineOilTemperature >= OIL_TEMP_VHIGH_LIMIT) {
+            setTempAmber(true);
+        } else {
+            setTempAmber(false);
+        }
+
+        if (engineOilTemperature > OIL_TEMP_HIGH_ADVISORY) {
+            setTimeout(() => {
+                if (engineOilTemperature > OIL_TEMP_HIGH_ADVISORY) {
+                    setTempBeenAboveAdvisory(true);
+                }
+            }, 900_000);
+        } else {
+            clearTimeout();
+            setTempBeenAboveAdvisory(false);
+        }
+
+        return () => clearTimeout();
+    }, [engineOilTemperature]);
+
+    useEffect(() => {
+        if (tempBeenAboveAdvisory) {
+            setTempAmber(true);
+        }
+    }, [tempBeenAboveAdvisory]);
+
+    useEffect(() => {
+        if (n2Percent >= 50) {
+            setIsValveOpen(false);
+        }
+    }, [n2Percent]);
+
+    useEffect(() => {
+        if (isEngineStarting && n2Percent < 50 && engSelectorPosition === 2) {
+            setTimeout(() => setIsValveOpen(true), 1200);
+        } else {
+            setTimeout(() => setIsValveOpen(false), 1200);
+        }
+
+        return () => clearTimeout();
+    }, [isEngineStarting, engSelectorPosition]);
+
+    useEffect(() => {
+        if (isEngineStarting && n2Igniting && n2Percent > 18 && n2Percent < 55) {
+            setShowIgniter(true);
+        } else {
+            setShowIgniter(false);
+        }
+    }, [isEngineStarting, n2Igniting, n2Percent]);
+
+    return (
+        <SvgGroup x={x} y={y}>
+            <text x={x} y={y} className="FillGreen FontLarge TextCenter">{displayedFuelUsed}</text>
+
+            <line className="GaugeMarking" x1={x - 51} y1={y + 85} x2={x - 45} y2={y + 85} />
+            <line className="GaugeMarking" x1={x} y1={y + 35} x2={x} y2={y + 40} />
+            <line className="GaugeMarking" x1={x + 45} y1={y + 85} x2={x + 51} y2={y + 85} />
```

Comment:
**thought:** We should adapt the `Mark` component to serve this function.

Reply:
The lines are more readable to me as opposed to using the Mark component.

Reply:
They're unreadable to me.

The Mark component manages rotation and thus it's clearer and only the necessary information is passed. It can be refactored as needed.